kyaml/yaml/merge3/multiplemergekeys_test.go

```diff
@@ -0,0 +1,422 @@
+package merge3_test
+
+var multipleMergeKeysTestCases = []testCase{
```

Comment:
doesn't this need to be added somewhere?

Reply:
oh I assumed these were examples I needed to add to my PR 

but if this PR is meant to be merged then I'll remove them from mine

Reply:
its fine to have distinct prs
just so all the tests go in first.
#3170 already merged